src/Entity/Committee.php

```diff
@@ -110,6 +111,11 @@ class Committee extends BaseGroup implements SynchronizedEntity
      */
     public $hosts = [];
 
+    /**
+     * @ORM\ManyToMany(targetEntity="AppBundle\Entity\ReferentTag")
```

Comment:
It's just a suggestion, but in an entity put `@var Collection|ReferentTag[]` doc is usefull when we want manipulate it in a custom method with auto-completion. Ide don't know yet read doctrine type as doctype ^^,